dev-support/Jenkinsfile

```diff
@@ -101,7 +98,8 @@ if [ ! -d "${YETUS_DIR}" ]; then
   gunzip -c yetus.tar.gz | tar xpf - -C "${YETUS_DIR}" --strip-components 1
 fi
         '''
-        dir ("${env.TOOLS}") {
+        // Set up the file we need at PERSONALITY_FILE location
+        dir ("tools") {
```

Comment:
This will always be relative to root? To $BASEDIR?

Reply:
it's relative to the workspace root.

Reply:
at the end of this stage we stash the tools directory and in future stages we unstash it relative to those stages workspace root.